code/controllers/autotransfer.dm

```diff
@@ -7,7 +7,7 @@ datum/controller/transfer_controller
 	var/shift_last_vote = 0 //VOREStation Edit
 datum/controller/transfer_controller/New()
 	timerbuffer = config.vote_autotransfer_initial
-	shift_hard_end = config.vote_autotransfer_initial + (config.vote_autotransfer_interval * 1) //VOREStation Edit //Change this "1" to how many extend votes you want there to be.
+	shift_hard_end = config.vote_autotransfer_initial + (config.vote_autotransfer_interval * 4) //VOREStation Edit //Change this "1" to how many extend votes you want there to be.
```

Comment:
One tiny thing, can you please tag this with "Mithra Edit" so if it gets updated down the line, we can Ctrl-F it and change it back easier?

Reply:
I'll resolve this on my side when I get the chance.

Reply:
Oh! Sorry I didn't even see the conversation start. I still am really new to Git processes. I'll be sure to look up how to tag for next time!

Reply:
All you need to do is change the comment to say `//Mithra Edit` instead of `//VOREStation Edit`, that'll be what we look for when we merge from upstream.